packages/eslint-config/package.json

```diff
@@ -19,30 +19,26 @@
     "url": "https://github.com/ueno-llc/eslint-config/issues"
   },
   "homepage": "https://github.com/ueno-llc/eslint-config#readme",
-  "devDependencies": {
-    "chai": "4.1.2",
-    "mocha": "3.5.3",
-    "xo": "0.18.2"
-  },
-  "engines": {
-    "node": ">=4"
-  },
-  "xo": {
-    "space": true
-  },
   "dependencies": {
     "babel-eslint": "8.0.1",
-    "deepmerge": "1.5.2",
+    "chai": "4.1.2",
     "eslint-config-airbnb": "16.0.0",
     "eslint-config-airbnb-base": "12.1.0",
     "eslint-import-resolver-webpack": "0.8.3",
     "eslint-plugin-flowtype": "2.36.0",
     "eslint-plugin-import": "2.7.0",
+    "eslint-plugin-internal": "file:rules",
```

Comment:
Not working at the moment. Installing the eslint-config through npm or via `yarn link` throw a `"/Users/jeremy/Ueno/starter-kit-universally/rules" doesn't exist`. The path seems to not be relative to the package it self.

Still looking into it

Reply:
What I tried so far:

- relative path as a dependency: `file:rules`
- absolute path to the rules folder
- postinstall script to install the rules folder as dependency